source/index.html

```diff
@@ -223,6 +223,12 @@ <h1>Home Assistant is sponsored by</h1>
       <div class="material-card">
         <h1>Home Assistant in the press</h1>
 
+        <a href="https://arstechnica.com/information-technology/2021/03/how-to-achieve-smart-home-nirvana-or-home-automation-without-subscription/"
```

Comment:
This should target _blank, and maybe also use noreferrer?

Reply:
I always mix them.... noopener is what I meant

Reply:
btw, 
![image](https://user-images.githubusercontent.com/1444314/113178415-69a65e00-9203-11eb-96c7-11c023891cfc.png)